4-projects/modules/infra_pipelines/variables.tf

```diff
@@ -14,6 +14,41 @@
  * limitations under the License.
  */
 
+variable "project_id" {
```

Comment:
We dont create a project within this module right?
https://github.com/terraform-google-modules/terraform-example-foundation/blob/2efcfd850d1c91dc76c3faa834f54f8120f6fb2a/4-projects/business_unit_1/shared/example_infra_pipeline.tf

Reply:
fixed